CleverFerret/src/main/java/com/universalmedialibrary/services/comic/ComicDataService.kt

```
@@ -2,22 +2,330 @@ package com.universalmedialibrary.services.comic
+        }
+    }
+    
+    /**
+     * Check if panel data exists for a comic
+     */
+    suspend fun hasPanelData(comicId: Long): Boolean {
+        return withContext(Dispatchers.IO) {
+            try {
+                val panels = comicPanelDao.getAllPanelsForComic(comicId)
+                panels.isNotEmpty()
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)